### DISCUSSION TITLE
tex文件

### DISCUSSION THREAD

**Discussion by Cyan99** [note286/xduts#28] [Q&A] [answered]
有点小白，真诚求问下：之前用的都是现成的模板。看这个xduts的pdf的意思是自己建main.tex文件，那摘要那些也是自己建立chapters文件夹，然后自己建abstract-zh.tex吗？我不知道我理解的对不对

**Comment by note286** [comment, marked as answer]
你理解的没有问题。

另外，`chapters`文件夹不是必要的，你都放在同一目录下或者其他目录里都是可以的，不命名为`abstract-zh.tex`也是可以的。不要教条，要动脑思考。

本项目的用法非常简单明了，再提供仅需几行即可编译的示例代码文件就有点侮辱用户智商了，故不提供示例代码文件。

**Reply by Cyan99** [reply]
好的好的，感谢哈  :)

**Comment by ahaaqiya** [comment]
楼主不好意思我想请问一下，按照您说的我在摘要那里直接引了路径，在写摘要的tex文件里只写了内容，但是编译以后最终pdf里关于摘要部分的中文全部是问号，，，应该是没有识别中文的意思，请问这是为什么呢，，，

**Reply by sikouhjw** [reply]
因为有『问号』，所以导入应该没问题。检查编码（统一 UTF-8）和编译方式（`xelatex`）是否正确。

**Reply by ahaaqiya** [reply]
唔，检查了，编码是UTF-8，编译方式是xelatex，我默认语言选的是en_US，不会是因为这个吧

**Reply by sikouhjw** [reply]
> 唔，检查了，编码是UTF-8，编译方式是xelatex，我默认语言选的是en_US，不会是因为这个吧

不会，不影响的。建议将 `main.tex` 和摘要文件的文字替换一下，上传上来

**Reply by ahaaqiya** [reply]
main.tex:
`\documentclass{xdupgthesis}
\usepackage{amsmath}
\usepackage{enumerate}
\usepackage{pifont}
\usepackage{theorem}
\usepackage{tabularray}
\usepackage{graphicx}
\usepackage{float}
\usepackage{diagbox}
\usepackage{subcaption}
\theoremheaderfont{\upshape\bfseries}
\theorembodyfont{\rmfamily}
\newtheorem{definition}{定义}[chapter]
\newtheorem{theorem}{定理}[chapter]
\renewcommand{\proofname}{\indent\bf 证明}
\usepackage{algorithm}
\usepackage[commentColor=black,beginComment=//~,beginLComment=//~, endLComment=]{algpseudocodex}
\tikzset{algpxIndentLine/.style={draw=black}}
\algrenewcommand{\algorithmicrequire}{\textbf{输入:}}
\algrenewcommand{\algorithmicensure}{\textbf{输出:}}
\usepackage{biblatex}
\addbibresource{sample.bib}
\usepackage{amsfonts,amssymb} 

\xdusetup{
	style = { cjk-font = fandol, latin-font = gyre, 
			  add-alg-rule-vspace=true,
		      ref-add-space = false},
	info = {
		title = {论如何让用户\\认真阅读文档},
		author = {张三},
		department = {排版学院},
		abstract = {chapters/abstract-zh.tex},		
                %abstracts* = {}
		keywords = {},
		keywords* = {Dummy,Keywords,Here,it is}
	}
}

\begin{document}
\begin{sloppypar}
	\chapter{绪论}
\end{sloppypar}
\end{document}`

abstract-zh.tex：
`这里是摘要`

显示是这样
![image](https://user-images.githubusercontent.com/91261133/226836642-80b12225-9411-4cc1-81e0-49ec6008c3a4.png)

**Reply by sikouhjw** [reply]
我的建议是：请把无关、重复宏包和环境删掉。

**Reply by ahaaqiya** [reply]
好的，，，，，确实看着有点乱了
